flixel/text/FlxBitmapText.hx

```diff
@@ -52,6 +52,7 @@ class FlxBitmapText extends FlxSprite
 	/**
 	 * Specifies how the text field should align text.
 	 * JUSTIFY alignment isn't supported.
+	 * Note: For alignment to work, 'autoSize' must be set to false.
```

Comment:
Yeah, we should be consistent about that and mention it in both places.

I would it a little differently though: "For alignment to have an effect [...]"

It's not that it doesn't work, you just wouldn't see it.